src/main/java/com/thealgorithms/conversions/OctalToBinary.java

```diff
@@ -0,0 +1,53 @@
+package com.thealgorithms.conversions;
+import java.util.Scanner;
+
+/**
+ * Converts any Binary Number to a Hexadecimal Number
+ *
+ * @author Bama Charan Chhandogi
+ */
+
+public class OctalToBinary {
+    public static void main(String[] args) {
+        Scanner scanner = new Scanner(System.in);
+        System.out.print("Enter an octal number: ");
+        int octalNumber = scanner.nextInt();
+
+        // Convert octal to binary
+        long binaryNumber = convertOctalToBinary(octalNumber);
+
+        System.out.println("Binary equivalent: " + binaryNumber);
+    }
+
```

Comment:
Please remove `main`, it's redundant

Reply:
All right, If I remove the main How to code will be run?
 I don't understand what is redundant

Reply:
This repository doesn't have an application that'd require an entry point. It's a library. If you want to call the provided functions, you can do so from your client code

Reply:
Please don't mind, I am just starting contributions to your repository. Can you please tell me what to do now? Just remove the main word.
thanks

Reply:
I've deleted the `main()` function